styles/choose-plan.css

```diff
@@ -1,9 +1,79 @@
 .choose-plan {
- margin-top: 126px;
+  margin-top: 126px;
 }
-.choose-plan__title{
-    text-align: center;
+.choose-plan__title {
+  text-align: center;
 }
-.choose-plan__description{
-    text-align: center;
+.choose-plan__description {
+  text-align: center;
+  margin: 30px 0;
+  font-weight: 500;
+  font-size: 18px;
+  line-height: 30px;
+}
+.choose-plan__wrap {
+  display: flex;
+  justify-content: space-between;
+  max-width: 1090px;
+  margin: 0 auto;
+}
+.choose-plan__item {
+  background: #ffffff;
+  border: 2px solid #dddddd;
+  box-sizing: border-box;
+  border-radius: 10px;
+  max-width: 330px;
+  width: 100%;
+  padding: 80px 0 50px 0;
+}
```

Comment:
It's a good thing to add side paddings also (left and right) — and you won't need horizontal margins for inner elements (such as a list).

Reply:
can i leave it like this? if i start fixing then centering of buttons and headers will have to be done margins..

Reply:
Leave it as it is for now then, let's discuss it offline tomorrow on our meeting :)